src/appendices/hed.md

```diff
@@ -1,73 +1,50 @@
 # Hierarchical Event Descriptors
 
-Hierarchical Event Descriptors (HED) are a controlled vocabulary of terms describing
-events in a machine-actionable form so that algorithms can use the information without
-manual recoding.
-HED annotation can be used to describe any experimental events by combining
-information from the dataset's `events.tsv` files and `events.json` sidecars.
+Hierarchical Event Descriptors (HED) are a controlled vocabulary of terms describing events in a machine-actionable form so that algorithms can use the information without manual recoding.
+HED annotation can be used to describe any experimental events by combining information from the dataset's `events.tsv` files and `events.json` sidecars.
```

Comment:
Please do not remove the line breaks. We (aspire to) use semantic breaking to reduce the size of diffs so that reviewers can focus only on changed lines in the specification.

https://github.com/bids-standard/bids-specification/blob/master/CONTRIBUTING.md#soft-rules

Reply:
I see, thank you for pointing that out.
How would it be best to fix that now?

Reply:
I would [`git checkout -p upstream/master`](https://git-scm.com/docs/git-checkout#Documentation/git-checkout.txt--p) and accept the patches for the sections you didn't edit.

Reply:
Thank you @effigies !
I have corrected it as suggested.